### PR TITLE
Feature/time slider

### PR DESCRIPTION
**Pull request by frasercl**
Adds a time slider (which doesn't have an associated issue?)

- The slider only appears for volumes with multiple time points
- Apart from spinboxes and extra styling (to come later), the slider appears as specified in #127
- While the slider is being dragged, the numeric readout updates but the new time point is not loaded
- When the slider handle is released, the new time point is loaded

### Screenshot:
image

### CODE REVIEW

File: src/aics-image-viewer/components/AxisClipSliders/index.tsx [hunk 1]
```diff
@@ -67,7 +111,7 @@ export default class AxisClipSliders extends React.Component<AxisClipSlidersProp
     const max = this.props.numSlices[activeAxis];
     const currentLeftSliderValue = Math.round(this.props.region[activeAxis][0] * max);
     const leftValue = (currentLeftSliderValue + delta + max) % max;
-    this.updateClipping(activeAxis, leftValue, leftValue);
+    this.updateClipping(activeAxis, leftValue, leftValue + 1);
```
Comment (toloudis) [inline src/aics-image-viewer/components/AxisClipSliders/index.tsx:114]: Confused about why this changed.  Is this resolving a bug in single slice mode?

File: src/aics-image-viewer/components/CellViewerCanvasWrapper/index.tsx [hunk 2]
```diff
@@ -17,9 +17,11 @@ interface ViewerWrapperProps {
   loadingImage: boolean;
   viewMode: ViewMode;
   appHeight: string;
-  image: Volume | null;
+  hasImage: boolean;
```
Comment (toloudis) [inline src/aics-image-viewer/components/CellViewerCanvasWrapper/index.tsx:20]: this is nice, a much less heavyweight dependency

**Review by toloudis — APPROVED** [review]
changes LGTM!  one minor question about the updateClipping calls.

File: src/aics-image-viewer/components/AxisClipSliders/index.tsx (hunk 1 above)
Comment (frasercl) [reply src/aics-image-viewer/components/AxisClipSliders/index.tsx:114]: Yes, this seems to be something I missed when changing this component during the functional refactor. Used to be that the min and max were offset by 1 automatically. Now they're not, so moving the slice with the step/play buttons would flatten it down to nothing. (Ought to take a look now or between milestones at whether slider numbers still make as much sense as they ought to.)

**Pull request merged**